server/__test__/index.test.js

```diff
@@ -416,4 +417,53 @@ describe('authentication', () => {
       });
     });
   });
+
+  describe('Test GET /api/v1/profile', () => {
+    test('should return status 200 and an object with correct data {message:"success",status:200,data:{}}', async () => {
+      const res = await request(app)
+        .get('/api/v1/profile')
+        .expect('Content-Type', /json/)
+        .set('Cookie', [`token=${token}`])
+        .expect(200);
+      const expected = {
+        status: 200,
+        message: 'success',
+        data: [
+          {
+            image:
+              'https://i.pinimg.com/564x/a9/f6/e3/a9f6e37a1211793bd2f45f161dc3dfbb.jpg',
+            firstname: 'zein',
+            lastname: 'jendeya',
+            email: 'zein@gmail.com',
+            weight: 63,
+            height: 1.68,
+            gender: 'f',
+            age: 19,
+          },
+        ],
+      };
+      const actual = res.body;
+      return expect(actual).toEqual(expected);
```

Comment:
This is very minor feedback, but the `actual` variable being created isn't really doing much, so you could just remove it and write `return expect(res.body).toEqual(expected);`

Reply:
I will pay attention to it
Thanks, Tom